web/src/main/java/org/cbioportal/web/util/ClinicalDataEqualityFilterApplier.java

```diff
@@ -29,18 +29,19 @@ public Integer apply(List<ClinicalDataEqualityFilter> attributes,
                          String studyId,
                          Boolean negateFilters) {
         Integer count = 0;
-
         for (ClinicalDataEqualityFilter s : attributes) {
             List<ClinicalData> entityClinicalData = (List<ClinicalData>)clinicalDataMap.get(entityId, studyId);
+            List<String> clinicalAttrs = s.getValues();
+            s.getValues().replaceAll(String::toUpperCase);
```

Comment:
I don't understand what you are doing here - what is this for?

Reply:
make filtering on clinical attributes case insensitive.